src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Colors [] colors = Colors.values();
```

Comment:
I wouldn't recommend making it an instance-level variable. Let's better move it to the getRandomColor() method.